script.js

```diff
@@ -0,0 +1,10 @@
+document.addEventListener("DOMContentLoaded", function() {
+    var projectBoxes = document.querySelectorAll('.project');
```

Comment:
hold off from using 'var' to declare variables - use 'let' or 'const' instead. as we've talked about in the past, var doesn't really obey scope rules that 'let' or 'const' do